### ISSUE TITLE
Has this been updated since the demo videos?

### ISSUE THREAD

**Issue by haveamission** [TheLX5/Cappy#1]
I've seen a lot of beta videos - has this been updated since then to give more interactivity with the world/control?

**Comment by TheLX5** [comment]
Not really. I kinda stopped since Nintendo took down one of Odyssey's videos. I really don't want the rest of my work be affected by continuing working on it.
